src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -114,8 +116,10 @@ public boolean isBrakeEngaged() {
         return isBrakeEngaged;
     }
 
+    public boolean isArmDown() { return armPosition == ArmPositions.DOWN; }
+
     public boolean isUP(){
-        return getAngle() >= 90;
+        return armPosition == ArmPositions.UP;
```

Comment:
The reason we were using a number instead of the enum is because it is possible for armPosition setpoint to be UP but the arm might still be moving. We could use the value of the enum though minus the tolerance and also change the method name from isUP to isUp since it isn't referring to the enum.